build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-  ext.kotlin_version = '1.4.21'
+  ext.kotlin_version = '1.4.20'
```

Comment:
Can you explain why we went from 1.4.21 to 1.4.20?

Reply:
Based on [this compatibility matrix](https://docs.gradle.org/6.8/userguide/compatibility.html) 1.4.20 is listed as the higher end so I think this small change is okay if it can provide more reliability

Reply:
Got it--thanks for clarifying!